### PR TITLE
Migrate Action Header to CSS Modules.

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Addresses issue #125

### CODE REVIEW

**Comment by mikertr** [comment]
Spacing issues in Chrome and Firefox.  Please see attached.  
chrome action header
firefox action header

File: packages/terra-clinical-action-header/src/ActionHeader.jsx [hunk 1]
```diff
@@ -100,13 +103,13 @@ const ActionHeader = ({
   const previousText = intl.formatMessage({ id: 'Terra.Clinical.ActionHeader.previous' });
   const nextText = intl.formatMessage({ id: 'Terra.Clinical.ActionHeader.next' });
 
-  const closeButton = onClose ? <Button icon={<IconClose ariaLabel={closeText} />} onClick={onClose} /> : null;
-  const backButton = onBack ? <Button icon={<IconLeft ariaLabel={backText} />} onClick={onBack} /> : null;
+  const closeButton = onClose ? <Button icon={<IconClose />} aria-label={closeText} onClick={onClose} /> : null;
+  const backButton = onBack ? <Button icon={<IconLeft />} aria-label={backText} onClick={onBack} /> : null;
```
Comment (Matt-Butler) [inline packages/terra-clinical-action-header/src/ActionHeader.jsx]: does the aria-label need to be on the icon?

**Comment by emilyrohrbough** [comment]
@mikertr Can you more detail on the issue you are seeing? The screenshots you provided seem identical to me, and they both appear to match what I see in chrome when running on the master branch: 
screen shot 2017-07-13 at 5 12 07 pm

I am wonder if this is a separate issue that needs to be logged.

**Comment by mikertr** [comment]
@emilyrohrbough I posted the screen shots with the working assumption that spacing between buttons and header title is incorrect.  By contrast, here it is in Safari.

screen shot 2017-07-14 at 8 51 21 am

**Comment by emilyrohrbough** [comment]
@mikertr Lets sync up today if possible. I was seeing the issues you described yesterday, but now when looking at the heroku instance, it appears to be working as expected.

**Comment by mikertr** [comment]
@emilyrohrbough and I looked at together this morning and are not seeing the spacing issues described yesterday.

**Comment by emilyrohrbough** [comment]
@mikertr does this mean this has been functionally verified and the needs functional review tag can be removed?

File: packages/terra-clinical-action-header/src/ActionHeader.jsx (hunk 1 above)
Comment (bjankord) [inline packages/terra-clinical-action-header/src/ActionHeader.jsx]: It should be on the button element, I believe screen readers have trouble announcing aria-label when its in the span or svg inside the button.

**Comment by PaulSolDev** [comment]
Functional review complete.

**Pull request merged**